src/Tickets/Seating/Service/Layouts.php

```diff
@@ -206,4 +208,26 @@ public static function update_transient_name(): string {
 	public static function update_transient_expiration() {
 		return 12 * HOUR_IN_SECONDS;
 	}
+	
+	/**
+	 * Returns the number of events associated with the layout.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $layout_id The ID of the layout.
+	 *
+	 * @return int The number of events associated with the layout.
+	 */
+	public static function get_associated_events_by_id( string $layout_id ): int {
+		try {
+			$count = DB::table( 'posts', 'posts' )
+						->leftJoin( 'postmeta', 'posts.id', 'layout_meta.post_id', 'layout_meta' )
```

Comment:
Should this the be the number of Events or Occurrences? 
Should this behaviour differ depending on CT1 being active or not?

Reply:
We don't have [support](https://lw.slack.com/archives/C05QHBFKYCA/p1718826994802359) for ASC tickets with recurring for now, so this should be good for the MVP.

Reply:
If occurrences of an recurring event is holding the same meta data as the parent event, then this query should take care of the occurrences as well. right?